packages/editor/src/components/block-settings-menu/reusable-block-convert-button.js

```diff
@@ -54,32 +54,39 @@ export default compose( [
 			canInsertBlockType,
 			__experimentalGetReusableBlock: getReusableBlock,
 		} = select( 'core/editor' );
+		const { canUser } = select( 'core' );
 
 		const blocks = getBlocksByClientId( clientIds );
 
-		const isVisible = (
-			// Hide 'Add to Reusable Blocks' when Reusable Blocks are disabled, i.e. when
-			// core/block is not in the allowed_block_types filter.
-			canInsertBlockType( 'core/block' ) &&
-
-			every( blocks, ( block ) => (
-				// Guard against the case where a regular block has *just* been converted to a
-				// reusable block and doesn't yet exist in the editor store.
-				!! block &&
-				// Only show the option to covert to reusable blocks on valid blocks.
-				block.isValid &&
-				// Make sure the block supports being converted into a reusable block (by default that is the case).
-				hasBlockSupport( block.name, 'reusable', true )
-			) )
+		const isReusable = (
+			blocks.length === 1 &&
+			blocks[ 0 ] &&
+			isReusableBlock( blocks[ 0 ] ) &&
+			!! getReusableBlock( blocks[ 0 ].attributes.ref )
 		);
 
 		return {
-			isVisible,
-			isStaticBlock: isVisible && (
-				blocks.length !== 1 ||
-				! isReusableBlock( blocks[ 0 ] ) ||
-				! getReusableBlock( blocks[ 0 ].attributes.ref )
+			// Show 'Convert to Regular Block' when selected block is a reusable block
+			isVisible: isReusable || (
+				// Hide 'Add to Reusable Blocks' when reusable blocks are disabled
+				canInsertBlockType( 'core/block' ) &&
+
+				every( blocks, ( block ) => (
+					// Guard against the case where a regular block has *just* been converted
+					!! block &&
```

Comment:
Again, any reason for the `!!`?

Reply:
It's to ensure that `typeof isVisible === 'boolean'`.